### DISCUSSION TITLE
Can I change my agents to point to an IP address instead of FQDN after server installation

### DISCUSSION THREAD

**Discussion by eagle856** [Ylianst/MeshCentral#4156] [Q&A] [answered]
I set up MeshCentral on an Oracle instance using Oracle-Linux. The install went fairly well considering I've never touched Linux before. When I set it up I used the path of _myoraclepath.oraclevcn.com_, which I assumed would resolve via DNS to my public IP. Come to find out it won't. Apparently Oracle does not publish their server for DNS resolution, so I was unable to connect my agents to the server. If I edit the MeshAgent.msh and replace the FQDN path with the public IP of the server, everything works. Since that file is created by the installer, is there a way to tell the installer to use an IP rather than the FQDN.

**Comment by Ylianst** [comment, marked as answer]
Duplicate of #4169

So yes, you can edit the config.json and in the "settings" section, do this:

```
{
  "settings": {
    "cert": "12.3.4"
   }
}
```

Once you restart the server, MeshCentral will use that IP address as the string all agents will need to connect to.

This said, I strongly recommend you get your own DNS name and point it to the IP address of your server, then do:

```
{
  "settings": {
    "cert": "sample.com"
   }
}
```

This is because IP addresses can change anytime and your going to be stuck re-installing all the agents. By using a DNS name, if you server changes IP address, you only need to update the DNS server.

Hope that helps.
